frontend/README.md

```diff
@@ -23,15 +24,16 @@ docker-compose up
 ```
 
 ## Without Docker
+
 Follow the steps below
 
-1. ```git clone https://github.com/SigNoz/signoz.git && cd signoz/frontend```
-1. change baseURL to ```<test environment URL>``` in file ```src/constants/env.ts```
```

Comment:
can you please remove this unnecessary change

Reply:
@palashgdev - Removed unnecessary changes & synced with upstream develop branch.